### PR TITLE
[W6.4h][T11-B2] Sarah Goh Shi Ning

### PR DESCRIPTION
**Pull request by sarahgoh97**
Printable Interface created for Name, Phone, Email, Address

### CODE REVIEW

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address implements Printable{
```
Comment (ashrafabdul) [inline src/seedu/addressbook/data/person/Address.java:9]: Coding standard violation. Missing whitespace before braces.

File: src/seedu/addressbook/data/person/Person.java [hunk 2]
```diff
@@ -84,4 +84,15 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public String getPrintableString(Printable... printables){
+        StringBuilder sb = new StringBuilder();
```
Comment (ashrafabdul) [inline src/seedu/addressbook/data/person/Person.java:91]: Good choice to use StringBuilder.

**Review by ashrafabdul — COMMENTED** [review]
@sarahgoh97 Added some comments. Please ack and close pr after reading.

**Comment by sarahgoh97** [comment]
Read. Thank you.

**Pull request closed without merging**